pkgs/development/ocaml-modules/cstruct/default.nix

```diff
@@ -1,20 +1,40 @@
-{stdenv, writeText, fetchurl, ocaml, ocplib-endian, sexplib_p4, findlib,
- async_p4 ? null, lwt ? null, camlp4}:
+{stdenv, buildOcaml, fetchFromGitHub, writeText,
+ ocaml, ocplib-endian, sexplib, findlib, ounit, camlp4,
```

Comment:
Yeah, I fixed it in the other PR, somehow I pull-requested the cstruct-update twice.